<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_connexion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>711</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion au serveur</string>
   </property>
   <widget class="QLabel" name="label_adresseIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adersse IP du serveur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num√©ro de port du serveur :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connexion">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adresse">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>172.18.58.104</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1025</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_etat">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lineEdit_etat">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_client">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>711</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Information Client</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>173</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_nomOrdi">
       <property name="text">
        <string>Nom de l'ordinateur distant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_nomUti">
       <property name="text">
        <string>Nom de l'utilisateur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_proco">
       <property name="text">
        <string>Architecture proceesseur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_os">
       <property name="text">
        <string>OS de l'ordonateur</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>491</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_nomOrdi"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_nomUti"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_proco"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_os"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_quitter</sender>
   <signal>clicked()</signal>
   <receiver>ClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
